src/NServiceBus.AcceptanceTesting/AcceptanceTestingContainer/AcceptanceTestingBuilder.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace NServiceBus
```

Comment:
Why do we need this?

Reply:
I was just following the

>For internal types, we also use the NServiceBus namespace. This allows unique identification of those types in logs and stack traces.

rule in https://github.com/Particular/NServiceBus/blob/develop/guidelines/design-guidelines.md#nservicebus-rules ?

Reply:
Sorry, my comment wasn't clear. What I meant was, why do we need `AcceptanceTestingBuilder` and `AcceptanceTestingContainer`?

Reply:
You mean to merge the container defintion with the IContainer implementation class?

Reply:
Think I get it now, https://github.com/Particular/NServiceBus/pull/5301/commits/261066e3014ca34d019ff8cc8c4bce4fe73de5c0 should address this.

Was this what you had in mind?

Reply:
While that does simplify things, so :+1: to that change, what I'm actually getting at is, why do we need a special acceptance testing version in the first place?